domain/build.gradle

```diff
@@ -4,5 +4,10 @@ dependencies {
 
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.3'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.3'
+    compile group: 'com.h2database', name: 'h2', version: '1.3.148'
+    compile group: 'org.hibernate.javax.persistence', name: 'hibernate-jpa-2.1-api', version: '1.0.0.Final'
+    compile 'org.springframework.boot:spring-boot-starter-web:2.0.1.RELEASE'
```

Comment:
Spring and database stuff shouldn't go inside the domain library.

Reply:
you should move this to the frontend package.

Reply:
when moved to the frontend package the service does not pick it up. Also alot of the annotations used in the domain package also stop working